src/main/scala/dpla/ingestion3/model/RowConverter.scala

```diff
@@ -0,0 +1,109 @@
+package dpla.ingestion3.model
+
+import dpla.ingestion3.model.DplaMapData.LiteralOrUri
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
+import org.apache.spark.sql.types.{ArrayType, StringType, StructField, StructType}
+
+object RowConverter {
+
+  def toRow(dplaMapData: DplaMapData, sqlSchema: StructType): Row =
+    new GenericRowWithSchema(
+      Array(dplaSourceResource(dplaMapData.sourceResource),
+        edmWebResource(dplaMapData.edmWebResource),
+        oreAggregation(dplaMapData.oreAggregation)),
+      sqlSchema
+    )
+
+  private[model] def oreAggregation(oa: OreAggregation): Row = Row(
+    oa.uri.toString,
+    edmAgent(oa.dataProvider),
+    oa.originalRecord,
+    oa.hasView.map(edmWebResource),
+    oa.intermediateProvider.map(edmAgent).orNull,
+    oa.`object`.map(edmWebResource).orNull,
+    oa.preview.map(edmWebResource).orNull,
+    edmAgent(oa.provider),
+    oa.edmRights.map(_.toString).orNull
+  )
+
+  private[model] def dplaSourceResource(sr: DplaSourceResource): Row = Row(
+    sr.alternateTitle,
```

Comment:
The order of fields really matters here, right? This ordering here determines the index value when converting from Row to Map? If this is the case I think a comment explaining that would be very helpful esp. when we need to update the model to add a field and maintain alphabetical order of field names.